packages/api-gateway/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM node:20.14.0-slim@sha256:5e8ac65a0231d76a388683d07ca36a9769ab019a85d85169fe28e206f7a3208e as build
+
+RUN corepack enable
+
+WORKDIR /app
+COPY package.json /app/
+COPY pnpm-lock.yaml /app/
+COPY pnpm-workspace.yaml /app/
+
+COPY ./packages/api-gateway/package.json /app/packages/api-gateway/package.json
+COPY ./packages/commons/package.json /app/packages/commons/package.json
```

Comment:
I think we need to add `api-clients` here.

Reply:
True! It seems to be missing also in BFF? 🤔 this means the build of BFF should be failing on main

Reply:
Seems to be because we did not add it to the `build_push_image:`, neither this API GW in this PR, nor the BFF! I could add both in this PR

Reply:
Ops 😄 , yep thank you.

Reply:
Done!